doc/asa.rst

```diff
@@ -3,6 +3,21 @@
 Security Advisory
 #################
 
+Addressed in ACRN v2.6
+************************
+
+We recommend that all developers upgrade to this v2.6 release (or later), which
+addresses the following security issues that were discovered in previous releases:
+
+-----
+
+-  Memory leakage vulnerability in ``devicemodel/hw/pci/xhci.c``
+    De-initializing of emulated USB devices result in memory leakage issue
```

Comment:
"result" --> "results"
"in memory leakage issue" --> "in a memory leakage issue" _or_ "in a memory leak"

Reply:
I fixed this and pushed the change back to yonghuah's branch